packages/web/src/components/range/MultiDropdownRange.js

```diff
@@ -220,9 +238,12 @@ class MultiDropdownRange extends Component {
 		const { value, onChange } = this.props;
 
 		if (value === undefined) {
-			this.selectItem(items);
+			this.selectItem({ item: items });
 		} else if (onChange) {
-			onChange(items);
+			this.selectItem({
```

Comment:
We don't need to create any side effect when controlled behavior is being used, Just pass the value to the `onChange`.